assets/responses/farmsmart_chat_ui_flow.json

```diff
@@ -0,0 +1,344 @@
+{
+  "uid": "1",
+  "title": "Chatbot",
+  "subtitle": "Welcome to FarmSmart chatbot!",
+  "origin": "com.wearemobilefirst.farmsmart",
+  "formResponse": {
+    "type": "com.google.firebase.database",
+    "uri": "profiles/%UserID/",
+    "body": {
+      "name": "%Name",
+      "skillLevel": "%SkillLevel",
+      "farmLocation": "%FarmLocation",
+      "agroEcoZone": "%AgroEcoZone",
+      "landSize": "%LandSize",
+      "motivation": "%Motivation",
+      "peopleInHousehold": "%PeopleInHousehold",
+      "soilType": "%SoilType",
+      "startDate": "%StartDate",
+      "usingIrrigation": "%UsingIrrigation",
+    }
+  },
+  "items": [
+    {
+      "text": "Hi! I'm the FarmSmart chatbot, Jata",
+      "sentiment": "Positive"
+    },
+    {
+      "text": "I'm here to set up your profile and recommend the best crops for your farm",
+      "sentiment": "Positive"
+    },
+    {
+      "text": "What is your name?",
+      "inputRequest": {
+        "type": "com.wearemobilefirst.String",
+        "uri": "%Name",
+        "localStore": false,
+        "inline": true,
+        "title": "Your Name",
```

Comment:
Not sure how we will translate titles or text that comes from JSON with `Intl.message` 🤔 We will have different JSON depending for language?

Reply:
yes we'll have a whole file per language (as this would normally be server side driven).